src/main/java/com/example/Library/services/MappingBookService.java

```diff
@@ -0,0 +1,9 @@
+package com.example.Library.services;
+
+import com.example.Library.dto.BookDTO;
+import com.example.Library.model.Book;
+
+public interface MappingBookService {
```

Comment:
there is no need to have MappingBookService interface, as this is simply a utility class with only one expected implementation. + Implementation could be renamed to 'BookMapper'